packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -1144,6 +1237,10 @@ export function TransferPanel() {
     if (isCctpTransfer) {
       return transferCctp()
     }
+    if (isDepositMode && nativeCurrency.isCustom && isSelectedTokenEther) {
```

Comment:
We can abstract this into it's own function which can be reused to identify the transfer case that is being implemented in these set of PRs.
`isEthTransferToCustomNativeCurrencyChain()`, and it's corresponding hook.

This flag can also be used in other places to show different UI case by case.